ios/FluentUI/Notification/MSFNotification.swift

```diff
@@ -196,6 +196,12 @@ import UIKit
     }
 
     @objc public static var allowsMultipleToasts: Bool = false
+
+    /// Only to be used when `isFlexibleWidthToast` is `true`
+    /// A set of constraints to be applied to the `MSFNotification`
+    /// in `show` after it has been added to the view hierarchy.
+    @objc public var flexibleWidthConstraints: [NSLayoutConstraint]?
```

Comment:
why do we need this at all? shouldn't clients add the constrainst themselves by grabbing the view?

Reply:
This is mostly so it's easier for them to use `.show` still. Otherwise I had to write a custom animation, or have the width constraint be a layout cycle late. I also ran into a crash when trying to show a second notification in a different window due to show being delayed while the first notification was dismissed.

Reply:
ok. so what was the issues to change line 107 to just equalTo rather than lessThanOrEqualTo ?

Reply:
Then it would be too long if the text was short. We want the width of the notification to hug the text.

Reply:
then your demo app changes will still have the smae issue?

Reply:
It would if the text didn't take up the whole space, yes. But, it's behaving how we wanted. And now we aren't setting any constraints internally to mess with that.

Reply:
having the nsconstraint var public in case another property is set to true by the client doesn't seem like the right design as a ux framework. either we do all the calculation correctly (figure out the intrinsic content size and update. the width anchor) or client code actually sets the width themselves is the much cleaner approach.